### PR TITLE
Java: Use 3rd party lib for version checking.

### PR DESCRIPTION
**Pull request by SanHalacogluImproving**
Issue: https://github.com/orgs/Bit-Quill/projects/4/views/2?itemId=55479660&sliceBy%5Bvalue%5D=Java+Milestone+IV&pane=issue

### CODE REVIEW

File: java/integTest/src/test/java/glide/SharedCommandTests.java [hunk 1]
```diff
@@ -776,7 +776,7 @@ public void expireAt_pexpireAt_and_ttl_with_positive_timeout(BaseClient client)
         assertTrue(client.ttl(key).get() <= 10L);
 
         // extend TTL
-        if (REDIS_VERSION.feature() < 7) {
+        if (REDIS_VERSION.isGreaterThanOrEqualTo("7.0.0")) {
```
Comment (Yury-Fridlyand) [inline java/integTest/src/test/java/glide/SharedCommandTests.java:779]:
```suggestion
        if (REDIS_VERSION.isLowerThan("7.0.0")) {
```

File: java/integTest/src/test/java/glide/SharedCommandTests.java [hunk 2]
```diff
@@ -789,7 +789,7 @@ public void expireAt_pexpireAt_and_ttl_with_positive_timeout(BaseClient client)
         }
         assertTrue(client.ttl(key).get() <= 50L);
 
-        if (REDIS_VERSION.feature() < 7) {
+        if (REDIS_VERSION.isGreaterThanOrEqualTo("7.0.0")) {
```
Comment (Yury-Fridlyand) [inline java/integTest/src/test/java/glide/SharedCommandTests.java:792]:
```suggestion
        if (REDIS_VERSION.isLowerThan("7.0.0")) {
```

**Review by Yury-Fridlyand — COMMENTED** [review]
Few tests fail, probably a typo

File: java/integTest/src/test/java/glide/SharedCommandTests.java (hunk 1 above)
Comment (SanHalacogluImproving) [reply java/integTest/src/test/java/glide/SharedCommandTests.java]: Fixed.

File: java/integTest/src/test/java/glide/SharedCommandTests.java (hunk 2 above)
Comment (SanHalacogluImproving) [reply java/integTest/src/test/java/glide/SharedCommandTests.java]: Fixed.

**Review by Yury-Fridlyand — APPROVED** [review]

**Pull request merged**